java/client/src/org/openqa/selenium/ie/InternetExplorerDriver.java

```diff
@@ -212,20 +219,6 @@ private InternetExplorerDriverService setupService(Capabilities caps, int port)
       builder.usingPort(port);
 
       if (caps != null) {
-        if (caps.getCapability(LOG_FILE) != null) {
-          String value = (String) caps.getCapability(LOG_FILE);
```

Comment:
If we can prepend "outputDir" in here to the LOG_FILE, we can then rely on the webdriver client to setup the appropriate logging. Right?

Reply:
The LOG_FILE is not the file path but the property "logFile" that the driver server checks for setting up the service. We can modify the caps to always set this particular property to the path we want for the log file but I don't see that approach as being too different from this one.